src/rust/catloop/mod.rs

```diff
@@ -121,8 +126,9 @@ impl CatloopLibOS {
 
     /// Instantiates a new LibOS.
     pub fn new() -> Self {
+        let mut rng: SmallRng = SmallRng::from_entropy();
```

Comment:
Could you add a configuration statement that would generate this from a fixed seed on debug mode?

Reply:
Done.